### DISCUSSION TITLE
How can I prevent xformers version rollback?

### DISCUSSION THREAD

**Discussion by Gazzoo-byte** [vladmandic/sdnext#524] [Q&A] [answered]
Trying to test newer versions of xformers however the newer versions get replaced with 0.0.17 regardless of what arguments I use, based on the console the arguments aren't checked until after xformers is checked

![image](https://user-images.githubusercontent.com/73721238/234633957-8eb21c53-55df-4d98-84f1-d59bef1c259c.png)

Same behaviour as above is noted when using the --experimental flag, and --skip-requirements, or any combination of the three - there doesn't seem to be a way to test alternate versions of xformers

*EDIT*

Following testing, xformers 0.0.19.dev524 actually leads to a slowdown of approximately 26% (~15it/s to ~11it/s on a RTX 4070) when compared to SDP

**Comment by vladmandic** [comment, marked as answer]
`set XFORMERS_PACKAGE="xformers==0.0.19"`
